kge/model/embedder/lookup_embedder.yaml

```diff
@@ -50,7 +50,9 @@ lookup_embedder:
     p: -1.                    # common choices: 1., 2.
 
   # Whether and how embeddings should be regularized. Possible values are '' (do
-  # not regularize) or lp (defaults to p=2; else set regularize_args.p)
+  # not regularize) or lp (defaults to p=2; else set regularize_args.p) or 
+  # n3 (which is l3 but the embedding vectors are treated as complex vector when
```

Comment:
which is l3 applied to the element-wise absolute value of complex embeddings)

Reply:
Also fixed ^^.